include/IMeshBuffer.h

```diff
@@ -573,8 +585,9 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
 	public:
 	    ICPUMeshBuffer(core::LeakDebugger* dbgr=NULL) : IMeshBuffer<core::ICPUBuffer>(NULL,dbgr), posAttrId(EVAI_ATTR0) {}
 
-
+		//! Returns id of position attribute.
```

Comment:
As mentioned in out previous conversations:

We only keep track of a position attribute, as every vertex needs to have at least a position to be displayed on the screen. Certain vertices may not have colors, normals, texture coords, etc. but a position is always present.

I wanted to give flexibility and not just assume attribute 0 to be the old gl_Vertex blindly.

On another note we need to know at least what is the vertex attribute for position to calculate bounding boxes reliably.

However if you want to do some weird GPGPU based off drawing vertices, you can safely ignore this if you don't use bounding box culling.